Jenkinsfile.public

```diff
@@ -289,7 +304,7 @@ pipeline {
                 stage('web/multinode') {
                     steps {
                         dir("web/multinode") {
-                            sh 'npm run lint-ci'
+                            sh 'npm run lint'
```

Comment:
this change is not needed

Reply:
This change was pulled in from the main repository when I updated my branch. I see it is still in main so I'm going to leave this as is for now. Once a modification to main is implemented, if needed, I will pull that change over.